Replace wwrelay-utils with pe-utils

mray190 opened the pull request:
Move in config and template files into snap-pelion-edge from wwrelay-utils
Remove wwrelay-utils part from the snap
Add in pe-utils part for identity.json generation

Note: Configuration files and templates have been moved around in these commits
DeviceDB is disabled to start, with a commit that we can revert later to turn back on

costanic commented on snap/snapcraft.yaml:
```diff
@@ -325,123 +325,17 @@ parts:
-        - $devtools
-        - $system
-        - $support
+        install ${SNAPCRAFT_PROJECT_DIR}/files/wwrelay-utils/version.json ${SNAPCRAFT_PART_INSTALL}/wigwag/etc/
```
since the part was renamed to pe-utils, we should move these files to a pe-utils folder.

mray190 replied:
Good catch - will fix that right now

mray190 commented:
Wait for v1.18.1 tag before merging to `dev`

mray190 commented:
Note that PR #276 was incorrectly merged. I rebased and removed that commit from this PR, and opened a new PR that we will keep closed for now. The icon says `merged` in this PR - that is NOT the case. Ignore it
